packages/client/wallets/aa/src/passkeys/services/CrossmintPasskeyService.ts

```diff
@@ -0,0 +1,43 @@
+import { BaseCrossmintService } from "@/api/BaseCrossmintService";
+import { PasskeyCipher } from "@/types";
+
+export class CrossmintPasskeyService extends BaseCrossmintService {
+    constructor(apiKey: string) {
+        super(apiKey);
+    }
+
```

Comment:
I'm pretty sure that if this constructor just calls `super` with the same param, we can omit the definition and it works the same
```suggestion
```

Reply:
nice one!